### PR TITLE
BB: Fix logs logic in stolen credentials analyzer

### PR DESCRIPTION
**Pull request by ordabach**
# What does this PR do?

Fix logs logic

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: envs/monkey_zoo/blackbox/analyzers/stolen_credentials_analyzer.py [hunk 1]
```diff
@@ -23,15 +23,13 @@ def analyze_test_results(self) -> bool:
...
             return True
-
-        if len(stolen_credentials) != len(self.expected_stolen_credentials):
+        else:
             self.log.add_entry(
-                f"Expected {len(self.expected_stolen_credentials)} credentials to be stolen but "
-                f"{len(stolen_credentials)} were stolen"
+                "The contents of the stolen credentials did not match the expected credentials"
             )
-        elif self.expected_stolen_credentials != stolen_credentials:
```
Comment (ilija-lazoroski) [inline envs/monkey_zoo/blackbox/analyzers/stolen_credentials_analyzer.py:32]: I don't think that it will ever get to this.

**Review by ilija-lazoroski — APPROVED** [review]
Issue and PR in the commit message and LGTM!

**Review by mssalvatore — COMMENTED** [review]
I'm not sure I understand this PR. It makes the log messages less specific. In the previous version, it told you about the number of stolen credentials if, and only if, they didn't match. And it told you about contents only if they didn't match. Now it always tells you both things, which means you need to read more log messages to figure out what went wrong.

**Comment by mssalvatore** [comment]
Closing this, as it makes the log messages more ambiguous.

**Pull request closed without merging**